packages/main/src/Menu.ts

```diff
@@ -375,6 +381,24 @@ class Menu extends UI5Element {
 			parentElement.selected = false;
 			(parentElement._popover.opener as HTMLElement)?.focus();
 		}
+		if (!menuItem.hasAttribute("ui5-menu-item")) {
+			return;
+		}
+		if (isUp(e)) {
+			this._handleNextOrPreviousItem(menuItem);
```

Comment:
Not sure if we should override the up and down interactions as those should come out of the box from the `ui5-list`.